src/changelog/2023/09/custom-node-support.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Custom Node Support
+date: 2023-09-28 12:01:01.0
+authors: ["marian-demme"]
```

Comment:
Maybe frontmatter should have a `doc_link: /docs/....` too?

Reply:
Fine with this being done next week, but I think nearly every item should end up with a link to the docs.